app/facades/food_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FoodFacade
+
+  def self.create_food_search(keyword)
+    response = FoodService.get_food_search(keyword)
+
+    response[:foods][0..9].map do |food_data|
```

Comment:
Consider using the API's query parameter of pageSize to limit results to 10 so that you don't have to use ruby here to do that (just a bit cleaner)